internal/services/cdn/cdn_frontdoor_route_disable_link_to_default_domain_resource.go

```diff
@@ -163,143 +164,173 @@ func resourceCdnFrontDoorRouteDisableLinkToDefaultDomainRead(d *pluginsdk.Resour
 	defer routeCancel()
 
 	customDomainClient := meta.(*clients.Client).Cdn.FrontDoorCustomDomainsClient
-	customDomainCtx, customDomaincancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
-	defer customDomaincancel()
+	customDomainCtx, customDomainCancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
+	defer customDomainCancel()
```

Comment:
this should be using the Read timeout from above

```suggestion
```

Reply:
Fixed.